### PR TITLE
Twitter example closes #408

### LINKED ISSUE #408: Add Twitter Example

### PR DESCRIPTION
**Pull request by kordjamshidi**

### CODE REVIEW

**Comment by kordjamshidi** [comment]
This is working.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .DS_Store
 data
-*/data
```
Comment (danyaljj) [inline .gitignore]: Why dropping this folder?

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/TwitterSentimentAnalysis/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
+# illinois-twitter-client
```
Comment (danyaljj) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/TwitterSentimentAnalysis/README.md]: I am not sure how this readme is relevant to this task. It just explains how to use someone else's twitter client, it doesn't say anything on the application. Maybe revise this readme with more relevant details? Like what is the input, what is the output, how to run, etc.

**Comment by danyaljj** [comment]
The PR needs an update with `IllinoisCogComp:master`.

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/TwitterSentimentAnalysis/README.md (hunk 2 above)
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/TwitterSentimentAnalysis/README.md]: Yes, this is todo today.

**Comment by bhargav** [comment]
Looks good! Can you also quickly `compile` the changes from the sbt console once so that the License headers are generated for the new files?


Okay to merge according to me. I'll let @danyaljj decide.

**Comment by kordjamshidi** [comment]
so merge please @danyaljj then

**Pull request merged**